charts/kafka/values.yaml

```diff
@@ -288,6 +291,23 @@ jmx:
   - kafka.network:*
   - kafka.log:*
 
+## volumePermissions: Change the owner of the persist volume mountpoint to RunAsUser:fsGroup
+##
+volumePermissions:
+  enabled: true
+  image:
+    registry: docker.io
+    repository: bitnami/minideb
+    tag: buster
```

Comment:
As above I think we can reduce the image size used to <1mb by using a busybox image